scripts/thrust_model_v2/Dynamic/generate_crazyflie_thrust_model.py

```diff
@@ -0,0 +1,128 @@
+import math
```

Comment:
Can you clean these files up a bit? there is commented out code and line spacing is all over the place.